lib/dndcolors.sty

```diff
@@ -0,0 +1,61 @@
+% Define colors, sampled from the books.
+\usepackage{color}
+
+% Page
+\definecolor{bgtan}{HTML}{F7F2E5}		  % background and quotebox
+\definecolor{pagegold}{HTML}{B89A67}  % pagenumbers and footer
+
+% Type
+\definecolor{titlered}{HTML}{58180D}	% titles
+\definecolor{titlegold}{HTML}{C9AD6A} % titlerules
+\definecolor{rulered}{HTML}{9C2B1B}		% triangular rule in statsblock
+
+% Trim (affects tables and paperboxes)
+\definecolor{PhbLightGreen}{HTML}{E0E5C1} % PHB Part 1
+\definecolor{PhbLightCyan}{HTML}{B5CEB8}  % PHB Part 2
+\definecolor{PhbMauve}{HTML}{DCCCC5}      % PHB Part 3
+\definecolor{PhbTan}{HTML}{E5D5AC}        % PHB appendix
+\definecolor{DmgLavender}{HTML}{E3CED3}   % DMG Part 1
+\definecolor{DmgCoral}{HTML}{F3D7C1}      % DMG Part 2
+\definecolor{DmgSlateGray}{HTML}{DBE4E4}  % DMG Part 3
+\definecolor{DmgLilac}{HTML}{D7D4D6}      % DMG appendix
+
+% Trim aliases
+\colorlet{DmgSlateGrey}{DmgSlateGray}
+
+% Stat blocks
+\definecolor{statblockribbon}{HTML}{E69A28} % top/bottom borders (gold)
+\definecolor{statblockbg}{HTML}{FDF1DC}     % stat block background (tan)
+
+% Global colour variables
+%
+%   commentboxcolor   commentbox background
+%   paperboxcolor     paperbox background
+%   trimcolor         colour of custom elements added by user
+%   tablecolor        table even row background
+%   themecolor        changing this with \setthemecolor sets all colours
+
+% Set the default theme to Part 1 of the PHB.
+\colorlet{themecolor}{PhbLightGreen}
+
+% Add element-specific colours.
+\colorlet{commentboxcolor}{themecolor}
+\colorlet{paperboxcolor}{themecolor}
+\colorlet{tablecolor}{themecolor}
+\colorlet{trimcolor}{themecolor}
+
+% Offer commands to set all accent colours.
+\newcommand{\setthemecolor}[1][themecolor]{%
+  \colorlet{commentboxcolor}{#1}
+  \colorlet{paperboxcolor}{#1}
+  \colorlet{tablecolor}{#1}
+  \colorlet{themecolor}{#1}
+  \colorlet{trimcolor}{#1}
+}
+\newcommand{\resetthemecolor}{\setthemecolor[PhbLightGreen]}
```

Comment:
I still think we should probably skip `\resetthemecolor`.

Reply:
On retrospect it might be confusing (i.e., does it set the default colour or the previous colour after changing it?). Removed.